### PR TITLE
[CWS][SEC-5162] report per rule metrics

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR reports per rule metrics specifying is the rule was loaded or ignored

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/metrics/metrics.go [hunk 1]
```diff
@@ -203,6 +203,9 @@ var (
 	// MetricPolicy is the name of the metric used to report policy versions
 	// Tags: -
 	MetricPolicy = newRuntimeMetric(".policy")
+	// MetricRuleStatus is the name of the metric used to report the rule status
+	// Tags: -
+	MetricRulesStatus = newRuntimeMetric(".rules_status")
```
Comment (lebauce) [inline pkg/security/metrics/metrics.go:208]: nit: rule_status ?

File: pkg/security/module/policy_monitor.go [hunk 2]
```diff
@@ -34,21 +37,37 @@ type Policy struct {
 	Version string
 }
 
+// IsIgnored defines a ignore state
```
Comment (lebauce) [inline pkg/security/module/policy_monitor.go:40]: nit: an ignore

**Review by lebauce — APPROVED** [review]
Just a few nits

File: pkg/security/metrics/metrics.go (hunk 1 above)
Comment (safchain) [reply pkg/security/metrics/metrics.go:208]: this metrics report the status of all the rule

**Pull request merged**